lua/nvim-gps/init.lua

```diff
@@ -52,6 +52,12 @@ local function setup_language_configs()
 				["string-name"] = ' '
 			}
 		}),
+		["latex"] = with_default_config({
+			icons = {
+				["text.title"] = "# ",
```

Comment:
Could you rename this to `text-title-name` or maybe just `title-name` and similarly for reference. Just so that its more in line with the naming convention for rest of the languages

Reply:
Done.